zerver/webhooks/taiga/view.py

```diff
@@ -101,45 +103,47 @@ def api_taiga_webhook(request: HttpRequest, user_profile: UserProfile,
         'unblocked': u':unlock: %(user)s unblocked task **%(subject)s**.',
         'set_milestone': u':calendar: %(user)s added task **%(subject)s** to sprint %(new)s.',
         'changed_milestone': u':calendar: %(user)s changed sprint of task '
                              '**%(subject)s** from %(old)s to %(new)s.',
-        'changed_status': u':chart_with_upwards_trend: %(user)s changed status of task **%(subject)s**'
+        'changed_status': u':chart: %(user)s changed status of task **%(subject)s**'
         ' from %(old)s to %(new)s.',
         'renamed': u':notebook: %(user)s renamed task %(old)s to **%(new)s**.',
         'description_diff': u':notebook: %(user)s updated description of task **%(subject)s**.',
-        'commented': u':thought_balloon: %(user)s commented on task **%(subject)s**.',
+        'commented': u':thought: %(user)s commented on task **%(subject)s**.',
         'delete': u':x: %(user)s deleted task **%(subject)s**.',
         'changed_us': u':clipboard: %(user)s moved task **%(subject)s** from user story %(old)s to %(new)s.'
     },
     'issue': {
         'create': u':bulb: %(user)s created issue **%(subject)s**.',
-        'set_assigned_to': u':busts_in_silhouette: %(user)s assigned issue **%(subject)s** to %(new)s.',
-        'unset_assigned_to': u':busts_in_silhouette: %(user)s unassigned issue **%(subject)s**.',
-        'changed_assigned_to': u':busts_in_silhouette: %(user)s reassigned issue **%(subject)s**'
+        'set_assigned_to': u':silhouettes: %(user)s assigned issue **%(subject)s** to %(new)s.',
+        'unset_assigned_to': u':silhouettes: %(user)s unassigned issue **%(subject)s**.',
+        'changed_assigned_to': u':silhouettes: %(user)s reassigned issue **%(subject)s**'
         ' from %(old)s to %(new)s.',
         'changed_priority': u':rocket: %(user)s changed priority of issue '
                             '**%(subject)s** from %(old)s to %(new)s.',
         'changed_severity': u':warning: %(user)s changed severity of issue '
                             '**%(subject)s** from %(old)s to %(new)s.',
-        'changed_status': u':chart_with_upwards_trend: %(user)s changed status of issue **%(subject)s**'
+        'changed_status': u':chart: %(user)s changed status of issue **%(subject)s**'
                            ' from %(old)s to %(new)s.',
         'changed_type': u':bulb: %(user)s changed type of issue **%(subject)s** from %(old)s to %(new)s.',
         'renamed': u':notebook: %(user)s renamed issue %(old)s to **%(new)s**.',
         'description_diff': u':notebook: %(user)s updated description of issue **%(subject)s**.',
-        'commented': u':thought_balloon: %(user)s commented on issue **%(subject)s**.',
+        'commented': u':thought: %(user)s commented on issue **%(subject)s**.',
         'delete': u':x: %(user)s deleted issue **%(subject)s**.'
     },
 }
 
 
 return_type = Tuple[Optional[Dict[str, Any]], Optional[Dict[str, Any]]]
+
```

Comment:
Please remove all of the extra newlines you added. If you want to make changes like this, it is better to move them to a separate commit. In any case, I don't think we need to be separating every code block with 2 newlines, since I am not really sure if we follow this guideline elsewhere in the code base, so better to keep the status quo. :)